package.json

```diff
@@ -44,7 +44,7 @@
     "regenerator-runtime": "^0.13.11",
     "register-service-worker": "^1.7.2",
     "sbc-common-components": "3.0.8",
-    "vue": "^2.7.14",
+    "vue": "^2.7.10",
```

Comment:
This won't work -- npm will see the `^` and automatically use the latest "2.x.x" version. See the package-lock.json file to see what version is actually used.

Also, after you change this, `rm -rf node_modules package-lock.json`, then `npm i`.

Reply:
Great catch. On it.

Reply:
This is resolved.